### PR TITLE
New icon added - ethereum

### PR DESCRIPTION
**Pull request by TecHMeaT**

### CODE REVIEW

File: images/svg/ethereum.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+xmlns:xlink="http://www.w3.org/1999/xlink"
+aria-label="workato" role="img"
```
Comment (edent) [inline images/svg/ethereum.svg]: Please correct the label. This is important for accessibility.

Comment (ghost) [reply images/svg/ethereum.svg]: Oh sorry, I was inattentive :)

**Comment by edent** [comment]
Thanks - you'll need to update the size on README as it's 1 byte larger now. Good work on this.

**Comment by TecHMeaT** [comment]
Thank you @edent, fixed

**Pull request merged**